README.md

```diff
@@ -1 +1,137 @@
+- Applied JavaScript best practices and language style guides in code.
+- Used JavaScript to manipulate DOM elements.
+- Used JavaScript events.
+- Used objects to store and access data.
+- Communicated technical concepts to other technical people.
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+
+Click [here] [https://alabi12.github.io/First-Captone-Project/] to watch a walk-through description, given by me.
+
+## Additional Tools
+
+- WebHint
+- Stylelint
+- ESlint
+- Node Packet Manager
+
+## Live Demo
+
+[Here] is the live demo.
+
+## Getting Started
+
+To get a local copy of this project up and running on your local machine follow the steps below.
+
+### Prerequisites
+
+- Basic html, css and medium Javascript know how.
+- Git installation
+- A text editor
+- A terminal
+- A web browser to view output e.g Google Chrome
+- An IDE e.g Visual studio code
+
+### Setup
+
+- Clone this repository or download the Zip folder:
+
+```
+git clone
+```
+
+- Navigate to the location of the folder in your machine:
+
+**`you@your-Pc-name:~$ cd <folder>`**
+
+### Usage
+
+- After Cloning this repo to your local machine
+- Open the `index.html` in your browser
+
+### Run Tests
+
+To track linter errors locally follow these steps:
+
+Download all the dependencies run:
+
+```
+npm install
+```
+
+Track HTML linter errors run:
+
+```
+npx hint .
+```
+
+Track CSS linter errors run:
+
+```
+npx stylelint "**/*.{css,scss}"
+```
+
+Track JavaScript linter errors run:
+
+```
+npx eslint .
+```
+
+### Deployment
+
+- I used GitHub Pages to deploy my website
+- For more information about publishing sources, see "[About github page](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#publishing-sources-for-github-pages-sites)"
+
+## Author
+
+👤 **Robert, Okoe Alabi**
+
+- GitHub: [@Alabi12](https://github.com/Alabi12)
+- Twitter: [@wolo_robert](https://twitter.com/wolo_robert)
+- LinkedIn: [Robert O. Alabi](https://linkedin.com/in/robert-o-alabi-49ba4b184)
+
+## Credits
+
+- The original design ideal by [Cindy Shin in Behance](https://www.behance.net/adagio07)
+- Project from [Microverse](https://bit.ly/MicroverseTN) html & css module
+- Images inspired by [Unsplash](unsplash.com)
+- Topic inspired by [Khadija Hamdani](https://www.researchgate.net/publication/329972741_Knowledge_based_entrepreneurship_the_role_of_networks)
+
+## Contributing
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/Alabi12/First-Captone-Project/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project and how we manage to build it!
+
+## Acknowledgments
+
+- Thanks to the Microverse team for the great curriculum.
+- Thanks to the Code Reviewer(s) for the insightful feedbacks.
+- A great thanks to My coding partner(s), morning session team, and standup team for their contributions.
+- Thanke to anyone whose code was used.
+
+## 📝 License
+
+This project is [MIT](MIT.md) licensed
```

Comment:
- Please include a loom video explaining and presenting your project, the video should not be more than 5 minutes long. 👍 👍